File: backend/generate_hash.py
import sys
import bcrypt


def main() -> None:
    if len(sys.argv) < 2:
        print('Usage: python generate_hash.py <password>')
        sys.exit(1)

    password = sys.argv[1]
    hashed = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt()).decode('utf-8')
    print(f'ADMIN_PASS_HASH={hashed}')


if __name__ == '__main__':
    main()

File: backend/generate-hash.js
const bcrypt = require('bcrypt');

const password = process.argv[2];
if (!password) {
  console.error('Usage: node generate-hash.js <password>');
  process.exit(1);
}

const hash = bcrypt.hashSync(password, 10);
console.log('BCRYPT_HASH=' + hash);
